pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -56,7 +57,7 @@ const (
 var (
 	extension = common.Extension{
 		Transformers: []mf.Transformer{ingress, egress, deploymentController, annotateAutoscalerService, augmentAutoscalerDeployment, addCaBundleToApiservice, configureLogURLTemplate},
-		PreInstalls:  []common.Extender{checkVersion, installNetworkPolicies, caBundleConfigMap},
+		PreInstalls:  []common.Extender{checkVersion, checkDependency, installNetworkPolicies, caBundleConfigMap},
```

Comment:
Can we maybe be more specific and call it `verifyServiceMeshMemberRoll` or something?